packages/integration-tests/projects/suite-web/tests/onboarding/t2-create-wallet.test.ts

```diff
@@ -11,44 +11,14 @@ describe('Onboarding - create wallet', () => {
 
     });
 
-    it('Success (no shamir capability)', () => {
```

Comment:
Why do we remove this one? Sorry we probably discussed it.

Reply:
That was done because the test is only valid for this old 2.1.4 firmware and cannot be used for any newer (bigger than 2.1.4) - and the same test scenario is below with `2-master`.

I think we did not discuss it, it was my decision to delete it, I thought it is no longer valid to test such old firmware. But we can revert it very quickly if we want

Reply:
Ah I see. It is over two years old which is the period when we stop supporting it. So let's keep it removed :).